### PR TITLE
added mine(ashis) and subham-rai github profile

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: ashissingh640@gmail.com

### CODE REVIEW

File: _staff_members/subham-rai.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 name: Subham Rai
 position: Contributor
 image_path: /images/photos/subham-rai.jpg
-# twitter: 
+twitter: https://github.com/subhamkrai
```
Comment (leecalcote) [inline _staff_members/subham-rai.md]: Needs changed to github

Comment (ashiscs) [reply _staff_members/subham-rai.md]: Ok sir @leecalcote

Comment (leecalcote) [inline _staff_members/subham-rai.md]: @gunjanmimo, I might be missing it, but I still see "twitter: " as the label used for what is a GitHub.com reference to @subhamkrai's profile. This needs changed to "github: ".

**Comment by ashiscs** [comment]
fixed this issue sir @leecalcote

**Pull request merged**